### ISSUE TITLE
for + plot() method error

### ISSUE THREAD

**Issue by No-longlong** [sfu-db/dataprep#152]
**Describe the bug**
just plot() method is not wrong. but with for() method, there isn't output.

```python
**for feature in ['Age', 'Cabin', 'Embarked']:
    plot(df, feature, 'Survived')**
```

**Expected behavior**
print plot

**Screenshots**

![image](https://user-images.githubusercontent.com/48305924/81663229-7c903280-9479-11ea-9e88-e2d0b5a2a807.png)

![image](https://user-images.githubusercontent.com/48305924/81663917-0809c380-947a-11ea-9214-beda480409b2.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome
 - Platform Jupyter Lab
 - Python Version [e.g. 3.0]
 - Dataprep Version [e.g. 0.2.3]

**Additional context**
(and why upgrading is doesn't work? it still be Ver 0.2.3. i think it's recently version is 0.2.5)

**Comment by jnwang** [comment]
Did you run **pip install -U dataprep** to upgrade?

**Comment by brandonlockhart** [comment]
Thanks for this great issue @No-longlong! Currently, plots will only render if the dataprep.eda function (or its returned object) is the last value in a cell.

@dovahcrow we identified this as a potential problem when we switched to using `_repr_html_`. One solution is to create support for pairs plots (#106) but with interactions with only one column (also similar to #139), which could give the desired result. Or, we could return to using Bokeh `show()` to render the plots in a for loop.
